### DISCUSSION TITLE
stories_things/cpp-things/bit/

### DISCUSSION THREAD

**Discussion by giscus** [Light-City/light-comments#13] [Announcements]
# stories_things/cpp-things/bit/

Stories About C Plus Plus

https://light-city.github.io/stories_things/cpp-things/bit/

**Comment by cxz0218** [comment]
大大，位域的对齐下面的学习代码learn.cpp跳转不见了，404，该在哪里查看呀~

**Reply by cxz0218** [reply]
我找到啦~

**Reply by cxz0218** [reply]
但是学习参考不可访问了，github上的地址同样不可访问

**Reply by Tlntin** [reply]
https://github.com/Light-City/CPlusPlusThings/blob/master/basic_content/bit/bit.cpp

**Comment by cxz0218** [comment]
作者大大，有一个小问题。我查到的资料显示，重映射通常是指将一个变量或数据结构的引用或指针重新指向不同的内存位置或对象，可以通过改变指针的指向或重新定义变量来实现。
”利用重映射将位域归零“的int* p = (int *) &b1 ，严格来说应该是强制类型转换叭（reinterpret_cast ），而不是严格意义的重映射？
